tests/js/chartAuthoringTests.js

```diff
@@ -78,15 +126,25 @@ https://github.com/floe/universal/LICENSE.txt
         fluid.each(that.templateLoader.resources, function (resource, resourceName) {
             jqUnit.assertValue("The resource text for " + resourceName + " should have been fetched", resource.resourceText);
         });
-        jqUnit.assertEquals("The dataEntryPanel has not been rendered", "", that.container.html());
+        jqUnit.assertEquals("The dataEntryPanel has not been rendered", null, that.dataEntryPanel);
+        jqUnit.assertEquals("The pieChart has not been rendered", null, that.pieChart);
```

Comment:
Line 129, 130 are to verify sub-components are instantiated instead of being rendered. Can you adjust test messages. Also, we can use ```jqUnit.assertNotUndefined``` to verify defined variables.